src/dmd/dtoh.d

```diff
@@ -235,6 +235,25 @@ extern(C++) void genCppHdrFiles(ref Modules ms)
     buf.writestring("#if !defined(_d_real)\n");
     buf.writestring("# define _d_real long double\n");
     buf.writestring("#endif\n");
+    buf.writeByte('\n');
```

Comment:
Can you do this as one large q-string please.

Reply:
Everything from the `#pragma once` to the end of this block of diff.

Reply:
I'm about to carve that whole block up and only emit fragments for the macros when they actually appear in the output.

Reply:
OK.